### PR TITLE
fix(codegen): Ignore import lines that may or may not be used 

### PR DESCRIPTION
**Pull request by ikoenigsknecht**

### CODE REVIEW

File: src/schema/SchemaGenerator.ts [hunk 1]
```diff
@@ -29,12 +29,16 @@ export class SchemaGenerator {
...
       import { GraphQLDateTime as DateTime } from 'graphql-iso-date';
+      // @ts-ignore
       import { GraphQLID as ID } from 'graphql';
+      // @ts-ignore
       import { ArgsType, Field as TypeGraphQLField, Float, InputType as TypeGraphQLInputType, Int } from 'type-graphql';
+      // @ts-ignore
       import { registerEnumType } from 'type-graphql';
 
-      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      // @ts-ignore eslint-disable-next-line @typescript-eslint/no-var-requires
```
Comment (goldcaddy77) [inline src/schema/SchemaGenerator.ts]: Do we need both of these?

Comment (ikoenigsknecht) [reply src/schema/SchemaGenerator.ts]: I think we do.

File: src/core/code-generator.ts [hunk 2]
```diff
@@ -30,6 +30,8 @@ export class CodeGenerator {
 
   constructor(
     private generatedFolder: string,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
```
Comment (goldcaddy77) [inline src/core/code-generator.ts]: What is the issue here?

File: src/test/examples/create-id-model/generated/classes.ts [hunk 3]
```diff
@@ -1,22 +1,14 @@
...
+  InputType as TypeGraphQLInputType
 } from "type-graphql";
 import { registerEnumType } from "type-graphql";
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const { GraphQLJSONObject } = require("graphql-type-json");
-
 import { BaseWhereInput, PaginationArgs } from "../../../../";
-// @ts-ignore
-import { CreateIDTest } from "../../create-id-model";
```
Comment (goldcaddy77) [inline src/test/examples/create-id-model/generated/classes.ts]: Did all of this get removed by the code generator?

**Comment by goldcaddy77** [comment]
If those changes to the test generated files were done by the code generator, good to :shipit:

File: src/core/code-generator.ts (hunk 2 above)
Comment (ikoenigsknecht) [reply src/core/code-generator.ts]: That field isn't used.

File: src/test/examples/create-id-model/generated/classes.ts (hunk 3 above)
Comment (ikoenigsknecht) [reply src/test/examples/create-id-model/generated/classes.ts]: This file doesn't appear to be auto-generated (not really sure how it got generated in the first place).  These changes were manual.  Happy to fix, though, if you can point me to where it actually comes from.

**Pull request merged**